sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateClient.cs

```diff
@@ -548,9 +583,11 @@ public virtual Response<byte[]> BackupCertificate(string certificateName, Cancel
         /// Creates a backup of the certificate, including all versions, which can be used to restore the certificate to the state at the time of the backup in the case the certificate is deleted, or to
         /// restore the certificate to a different vault in the same region as the original value. This operation requires the certificate/backup permission.
         /// </summary>
-        /// <param name="certificateName">The name of the <see cref="KeyVaultCertificate"/> to backup</param>
+        /// <param name="certificateName">The name of the <see cref="KeyVaultCertificate"/> to backup.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
-        /// <returns>The certificate backup</returns>
+        /// <returns>The certificate backup.</returns>
+        /// <exception cref="ArgumentException"><paramref name="certificateName"/> is empty.</exception>
```

Comment:
Out of curiosity, what is intended to go between <exception></exception> blocks?  Is it the message the exception contains?  Nit-picky me is concerned that this sentence doesn't start with a capital, although I understand why.

Reply:
Not typically. If you look at .NET's documentation (Framework and Core), it an explanation of *why* the exception would be thrown like this. In fact, they use `<paramref>` in this manner a lot in those statements.